docs/aerodromes/classd/Moorabbin.md

```diff
@@ -8,9 +8,23 @@
 | Name               | Callsign       | Frequency        | Login ID                         |
 | ------------------ | -------------- | ---------------- | ---------------------------------------- |
 | **Moorabbin ADC**  | **Moorabbin Tower**  | **118.100**         | **MB_TWR**                        |
+| Moorabbin ADC 2†  | Moorabbin Tower  | 123.000        | MB-2_TWR                        |
 | **Moorabbin SMC**   | **Moorabbin Ground**   | **134.250**          | **MB_GND**                       |
 | Moorabbin ATIS        |                | 120.900          | YMMB_ATIS                               |
 
+† *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies){target=new}
+
+### Dual ADC Operations
+When Moorabbin ADC 2 is online, responsibility for the **Runway**, **Circuit**, and **Airspace** is divided between the two ADC controllers.
```

Comment:
Do you want me to integrate this into the PR I been doing for MB

Reply:
Nah no need, we'll manage this separately